src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import CommentInfo from '../CommentInfo/index';
```

Comment:
No need to write index here. Index will automatically be used if you import from folder. We create this file to shorten import code.
```suggestion
import React from 'react';

import CommentInfo from '../CommentInfo';
```